libsystemd-journal became libsystemd, and the compatibility package doesn't
seem to work for us.

Index: libsystemd-journal-1.2.0/libsystemd-journal.cabal
===================================================================
--- libsystemd-journal-1.2.0.orig/libsystemd-journal.cabal	2014-12-21 23:06:51.397522808 +0100
+++ libsystemd-journal-1.2.0/libsystemd-journal.cabal	2014-12-21 23:07:05.257798150 +0100
@@ -19,5 +19,5 @@
   build-depends:       base >=4.6 && <4.8, bytestring >= 0.9.1, pipes >= 4.0, pipes-safe >= 2.0, text >= 0.1 && < 1.2, transformers >= 0.3, unix-bytestring >= 0.3.2 && < 0.4, vector >= 0.4 && < 0.11, uuid, unordered-containers >= 0.1 && < 0.3, hashable >= 1.1.2.5, hsyslog, uniplate >= 1.6
   hs-source-dirs:      src
   default-language:    Haskell2010
-  extra-libraries: systemd-journal
+  extra-libraries: systemd
   ghc-options: -Wall
